appinventor/components/src/com/google/appinventor/components/scripts/ComponentProcessor.java

```diff
@@ -423,7 +423,7 @@ public void setDescription(String description) {
       } else {
         // Throw out the first @ or { and everything after it,
         // in order to strip out @param, @author, {@link ...}, etc.
-        this.description = description.split("[@{]")[0].trim();
+        this.description = description.split("[@|{@]+(([a-zA-Z])*)+[(\\s)*]")[0].trim();
```

Comment:
This seems to test correctly. I'd like to ask another dev to take a look as a sanity check.